adapters/IronSource/IronSourceAdapter/GADMAdapterIronSourceRewardedAd.m

```diff
@@ -1,4 +1,4 @@
-// Copyright 2019 Google Inc.
+// Copyright 2023 Google Inc.
```

Comment:
Revert all the changes for the years if the files aren't added in this PR.

Reply:
Done